project/src/com/company/entities/Account.java

```diff
@@ -0,0 +1,103 @@
+package com.company.entities;
+
+import java.util.Vector;
+
+public abstract class Account implements Comparable<Account>{
+    String first_name;
+    String last_name;
+    String email;
+
+    Vector<Card> cards; //an account can have multiple cards
+    Savings savings; //an account can have savings
+    Vector<Transaction> transactions; //here we store all the transactions associated with the account
+
+    //constructor
+    public Account(String first_name, String last_name, String email) {
+        this.first_name = first_name;
+        this.last_name = last_name;
+        this.email = email;
+
+        this.cards = new Vector<Card>();
+        this.savings = new Savings(this);
+        this.transactions = new Vector<Transaction>();
```

Comment:
remove Card and Transaction from type of Vector, use diamond operator: <>

Reply:
Fixed.